build.gradle

```diff
@@ -86,7 +86,12 @@ publishing {
                         }
                     }
                 }
-
+                developers {
```

Comment:
Is this really necessary?  I don't care about credit per se.

Reply:
It seems that Bintray fails at POM validation stage due to missing 'developers' section.

Reply:
Good answer.